common.yml

```diff
@@ -10,6 +10,8 @@
   roles:
     - greendayonfire.mongodb
 
+- include: tsuru_graphite.yml
+
```

Comment:
I just put it there because there was a comment about bringing up DB tier. Metrics and graphite are kind-of data. But it can be moved anywhere, it's just a line...